scapy/contrib/uds.py

```diff
@@ -0,0 +1,977 @@
+#! /usr/bin/env python
+
+# This file is part of Scapy
+# See http://www.secdev.org/projects/scapy for more informations
+# Copyright (C) Nils Weiss <nils@we155.de>
+# This program is published under a GPLv2 license
+
+from scapy.fields import ByteEnumField, StrField, ConditionalField, \
+    BitEnumField, BitField, XByteField, \
+    XShortField, X3BytesField, XIntField, ByteField, \
+    ShortField, ObservableDict, XShortEnumField, XByteEnumField
+from scapy.packet import Packet, bind_layers
+import struct
```

Comment:
That's a detail but please move this import before the scapy imports.

The rule is simple: first built-in module imports, then external modules, then internal submodules.

Reply:
Done